### DISCUSSION TITLE
About security

### DISCUSSION THREAD

**Discussion by heeyeonkoo99** [PaddlePaddle/PaddleOCR#12762] [Q&A] [answered]
Hello, I'm a student looking to use PaddleOCR for a project. I'm just wondering if it's okay to use this framework without worrying about data leakage in terms of security?

**Comment by SWHL** [comment]
What do you mean about data leakage in terms of securiy?

**Comment by heeyeonkoo99** [comment]
I am working on a project that involves handling sensitive company data, and I am wondering if I should be concerned about the risk of data leakage.

-----Original Message-----

**Comment by GreatV** [comment, marked as answer]
You can run this project in offline mode.

**Comment by heeyeonkoo99** [comment]
Thank you for replying!

-----Original Message-----
